src/website/shared/github.py

```diff
@@ -0,0 +1,97 @@
+import logging
```

Comment:
Up to `create_gh_issue`, this file hasn't changed - I'm not sure why `git` didn't pick it as a move + additions.